testsuite/tests/win-unicode/Makefile

```diff
@@ -9,7 +9,7 @@ C_FILES=mkfiles
 .PHONY: test
 test:
 	@if echo 'let () = exit (if Config.windows_unicode then 0 else 1)' | $(OCAML) -I $(OTOPDIR)/utils config.cmo -stdin; then \
-	  $(MAKE) printargv.exe printenv.exe symlink_tests.precheck && \
+	  $(MAKE) printargv.exe printenv.exe symlink_tests.precheck exec_tests.precheck && \
```

Comment:
This seems a little convoluted to me - why not just create exec_tests.precheck with `exit 1` in it - the entry in .gitignore is then not necessary and lines 22-25 below can just be deleted?

Reply:
The `exec_tests.precheck` file could also then include a comment as to why the tests are disabled (stability - which I will look into, as I have an idea what it is...)

Reply:
Fixed as suggested, thanks!